### ISSUE TITLE
Abandoned?

### ISSUE THREAD

**Issue by csvan** [alexk111/ngImgCrop#83]
It is close to a year since the last commit, and there are several PRs hanging in the ether. Is this project abandoned, and if so is anyone up for collaborating on a fork?

**Comment by Bogatinovski** [comment]
Also I have a question, is the rectangular crop implemented? Thanks!

**Comment by Jesspu** [comment]
I would like to know as well. I would love to use this for a in-house app I am making for a company but I am curious if there is still any work being done here?

**Comment by Bogatinovski** [comment]
In the mean time I started using this plugin.
https://github.com/AllanBishop/angular-img-cropper
It has support for rectangular resize with aspect-ratio and does the job very well.

**Comment by Jesspu** [comment]
Yeah I ended up finding this one, just swapped out the files and it worked perfectly https://github.com/jodonnell-broadsoft/JsImageCrop
